client/k8s.go

```diff
@@ -19,12 +19,12 @@ package client
 import (
 	"context"
 	"flag"
-	"os"
-	"path/filepath"
-
 	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/storage/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os"
+	"path/filepath"
+	"strings"
```

Comment:
^ Ignore this change suggestion

I need to validate if this actually works correctly in my IDE, ideally goimports should have failed the CI here. I'll create a `good-first-issue` for ci revamp and further aggressive linting for new contributors. This can be documented as devel docs, etc.

Reply:
- I run gofmt, golint and goimports everytime before checking in. Not sure, why it doesn't work at times. I do it from the cli and not Goland, it should have worked.